TwitchLib.PubSub/Events/OnWhisperArgs.cs

```diff
@@ -2,10 +2,46 @@
 
 namespace TwitchLib.PubSub.Events
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Drawing;
+    using Models.Responses.Messages.Whispers;
+
     /// <summary>[INCOMPLETE/NOT_FULLY_SUPPORTED]Whisper arguement class.</summary>
     public class OnWhisperArgs
     {
-        /// <summary>Property representing the whisper object.</summary>
+        /// <summary>TOOOOOOOOOOOOODOOOOOOOOO</summary>
```

Comment:
The todo is referring to the DateTime object for `SentTimestamp` right? Is it broken at the moment?

Reply:
Even though I added a TOOOOOOOODOOOOO in full caps, I missed it D: Was replacing all Todo's with correct summaries. Forgot that one.. *sighs*